package.nls.json

```diff
@@ -432,6 +432,7 @@
     "AWS.samcli.deploy.s3bucket.picker.loading": "Loading S3 buckets...",
     "AWS.samcli.deploy.s3bucket.picker.noBuckets": "No buckets found.",
     "AWS.samcli.deploy.s3bucket.picker.error": "There was an error loading S3 buckets.",
+    "AWS.samcli.deploy.s3bucket.picker.createNewBucket": "Create New S3 Bucket...",
```

Comment:
Thinking you can just use `AWS.command.s3.createBucket`

Reply:
Done